Package.swift

```diff
@@ -17,7 +17,8 @@ var apodiniSwiftSettings: [SwiftSetting] {
             .unsafeFlags(
                 [
                     "-Xfrontend",
-                    "-enable-experimental-concurrency"
+                    "-enable-experimental-concurrency",
+                    "-DAPODINI_EXPERIMENTAL_ASYNC_AWAIT"
```

Comment:
Something not entirely related, but maybe something we should discuss as we are seemingly introducing more and more custom Active Compilation Conditions (like in this PR with `APODINI_EXPERIMENTAL_ASYNC_AWAIT`).
In order to optimize build time of our CI suite, we somehow need to match compiler arguments for those test suites which do custom compilation steps (this are ApodiniDeploy and NegativeCompileTests at the moment; See [Note to CI maintainers](https://github.com/Apodini/Apodini/blob/develop/Documentation/Testing/NegativeCompileTests.md#note-to-ci-maintainers)).
My current approach was to manually hardcode those (see [here](https://github.com/Apodini/Apodini/blob/1183e58ebdeba14959ae3840e88cf7853c93f8e1/Tests/NegativeCompileTestsRunner/NegativeTestRunner.swift#L268-L285)) as those shouldn't change that frequently, but I can see where such a change may go unnoticed in the future. Maybe we can find a way to automatically/dynamically grab the compiler arguments passed to the compiler?

Anyways, as we currently don't compile tests with async await enabled this isn't a problem. Though, maybe it makes sense to add support for the `APODINI_EXPERIMENTAL_ASYNC_AWAIT` in the NegativeTestRunner already(?).